### ISSUE TITLE
Not able to save dataset draft

### ISSUE THREAD

**Issue by philippconzett** [DataverseNO/docker.dataverse.no#35]
Was not able to new dataset draft on dataverse.no on April 5, 2023 at 23:46 CEST. @rolfande Maybe a firewall rule configuration issue?

**Comment by rolfande** [comment]
@philippconzett Can you check if this also was solved after the reconfiguration related to #34 ?

**Comment by philippconzett** [comment]
Yes, thanks. I forgot to mention that the two issues were related.
